src/modules/SX127x/SX127x.cpp

```diff
@@ -782,6 +782,24 @@ int16_t SX127x::setPreambleLength(size_t preambleLength) {
   return(RADIOLIB_ERR_UNKNOWN);
 }
 
+int16_t SX127x::setPreamblePolarity(int polarity) {
```

Comment:
Since there are only two possible values, I think it would be better to use a `bool` polarity, with `true` using the default and false for the inverted.

Reply:
If the value is just true or false how can you tell which one is 0x55 or 0xAA? There is no standard preamble in the FSK world.

Reply:
It's true there is no standard in general - what I meant by default is the current polarity, which is fixed at 0x55 (sorry for not being clear).

Reply:
Do you think it would be more convenient having a setPreamblePolarity55(true|false) method instead of the one above?

Reply:
I think it's fine with the current name. Maybe `invertPreamble(true|false);` could be more descriptive, analogous to `SX127x::invertIQ(true|false)`?